### ISSUE TITLE
al_get_standard_path() flag to get OS X app bundle path

### ISSUE THREAD

**Issue by fatcerberus** [liballeg/allegro5#700]
Currently Allegro has `ALLEGRO_RESOURCE_PATH` and `ALLEGRO_EXENAME_PATH`.  It would be nice to have, e.g. `ALLEGRO_APPLICATION_PATH` which gives the path to either the .app bundle (for bundled apps) or the actual executable (everything else).  In order to do this generically, one has to first determine whether the app is bundled (and then *only* do that on OS X), but Allegro already has that detection built in, so it'd be easier to have Allegro do it.

**Comment by EdgarReynaldo** [comment]
This would be nice as we're currently having an issue trying to locate allegro.log on OSX on this thread here : https://www.allegro.cc/forums/thread/617458/1038184#target

**Comment by fatcerberus** [comment]
Yeah, the thing is that for bundled apps the path returned for EXENAME is for the bundled executable and not the `.app` folder itself.  That executable is about 3 subdirectories down.
